staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -68,29 +69,39 @@ const (
 )
 
 // NewLeaderElector creates a LeaderElector from a LeaderElectionConfig
-func NewLeaderElector(lec LeaderElectionConfig) (*LeaderElector, error) {
+func NewLeaderElector(lec LeaderElectionConfig) (*LeaderElector, []error) {
```

Comment:
is staging/src/k8s.io/client-go/ the right place for making changes to client-go  since its generated ?

Reply:
It was once moved from pkg/client.